test/mocks/seed.ts

```
@@ -192,14 +229,97 @@ const processItemMemberships = (items: DataType['items'] = []) => {
+  members?: SeedMember[];
+}) {
+  // get all unique members
+  const allMembers = [
+    ...members,
+    ...items.flatMap((i) => {
+      // get all account from all memberships
+      const accountsFromMemberships = (i.memberships ?? [])?.reduce((acc, m) => {
+        if (!m.account) {
+          return acc;
+        }
+        return [...acc, m.account];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)